target/linux/ramips/dts/BARIONET-1000-V1-1-A9.dts

```diff
@@ -0,0 +1,145 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "BARIONET-1000-V1-1-A9", "ralink,mt7620a-soc";
+	model = "Barix Barionet 1000 v1.1.A9";
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+	aliases {
+		spi0 = &spi0;
+		spi1 = &spi1;
+		serial0 = &uartlite;
+		serial1 = &uart;
+	};
+	gpio-leds {
+		compatible = "gpio-leds";
+		link {
+			label = "barionet1000_eth_link";
+			gpios = <&gpio2 0 1>;
+		};
+		data {
+			label = "barionet1000_eth_data";
+			gpios = <&gpio2 4 1>;
+		};
+	};
+};
+
+&uart {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&gpio3 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x30000>;
+			read-only;
+		};
+
+		partition@30000 {
+			label = "u-boot-env";
+			reg = <0x30000 0x10000>;
+			read-only;
+		};
+
+		factory: partition@40000 {
+			label = "factory";
+			reg = <0x40000 0x10000>;
+			read-only;
+		};
+
+		partition@50000 {
+			label = "firmware";
+			reg = <0x50000 0xfb0000>;
+		};
+	};
+};
+
+&spi1 {
+	status = "okay";
+	mcp3x0x@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		reg = <0 0>;
+		compatible = "mcp3204";
```

Comment:
According to http://elixir.free-electrons.com/linux/v4.9.34/source/Documentation/devicetree/bindings/iio/adc/mcp320x.txt  just `mcp3204` is deprecated. Please use `microchip,mcp3204`.